### PR TITLE
Adding calculation of main elements with the web browser bar

### PR DESCRIPTION
**Pull request by pkrucz00**

### CODE REVIEW

**Comment by pkrucz00** [comment]
There is a new feature `dvw`. I need to check that

File: assets/js/components/Router.tsx [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
 import Recording from "../pages/Recording";
 
 const Router = () => {
+  useEffect(() => {
+    const changeCssHeightVariable = () =>
+      window.document.documentElement.style.setProperty("--window-inner-height", `${window.innerHeight}px`);
+    changeCssHeightVariable();
+    window.addEventListener("resize", changeCssHeightVariable);
+
+    return window.removeEventListener("resize", changeCssHeightVariable);
```
Comment (Rados13) [inline assets/js/components/Router.tsx:14]: Shouldn't it be a function?

Comment (pkrucz00) [reply assets/js/components/Router.tsx]: Should be. Changed that

**Comment by pkrucz00** [comment]
Pushed `ML-157-presenter-on-android` branch here by accident. Reopening

**Pull request merged**